core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -265,10 +265,6 @@
 			'inputType'               => 'imageSize',
 			'reference'               => &$GLOBALS['TL_LANG']['MSC'],
 			'eval'                    => array('rgxp'=>'natural', 'includeBlankOption'=>true, 'nospace'=>true, 'helpwizard'=>true, 'tl_class'=>'w50'),
-			'options_callback' => static function ()
```

Comment:
Why did you remove the options callback here (and in tl_layout and tl_module)?

Reply:
Because I added a [general purpose listener for this and use it via annotations](https://github.com/contao/contao/pull/3630/files#diff-d71a0a24c80180705e7fd49548488f67bc086cb5cc0e547dea38794fb68428f5R21-R23)